### ISSUE TITLE
download_MR_data.sh re-downloads PTB_ACRPhantom_GRAPPA.zip

### ISSUE THREAD

**Issue by KrisThielemans** [CCPPETMR/SIRF-Exercises#21]
running `download_MR_data.sh` twice results in 
```
grappa2_6rep.h5: OK
File exists and its md5sum is ok
--2019-04-11 08:05:12--  https://zenodo.org/record/2633785/files/PTB_ACRPhantom_GRAPPA.zip
Resolving zenodo.org (zenodo.org)... 137.138.76.77
Connecting to zenodo.org (zenodo.org)|137.138.76.77|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 6401739 (6.1M) [application/octet-stream]
Saving to: ‘PTB_ACRPhantom_GRAPPA.zip.1’

PTB_ACRPhantom_GRAP 100%[===================>]   6.10M  4.05MB/s    in 1.5s    

2019-04-11 08:05:14 (4.05 MB/s) - ‘PTB_ACRPhantom_GRAPPA.zip.1’ saved [6401739/6401739]

Unpacking PTB_ACRPhantom_GRAPPA.zip
Archive:  PTB_ACRPhantom_GRAPPA.zip
replace PTB_ACRPhantom_GRAPPA/ptb_resolutionphantom_fully_ismrmrd.h5? [y]es, [n]o, [A]ll, [N]one, [r]ename: 
```
The script should probably use the `download` function, like the others

**Comment by johannesmayer** [comment]
Yeah the `download` function has these `if` loops asking about `md5sum` of the file but that fails for the dataset I uploaded to Zenodo. Unforunately I didn't know I had to add any of those to the dataset.

**Comment by KrisThielemans** [comment]
First compute it:
```
$ md5sum PTB_ACRPhantom_GRAPPA.zip 
a7e0b72a964b1e84d37f9609acd77ef2  PTB_ACRPhantom_GRAPPA.zip
```
Then update the script.

**Comment by KrisThielemans** [comment]
great! Please update https://github.com/CCPPETMR/SIRF-Exercises/blob/master/INSTALL.md as well? Thanks!

@paskino, due to our current `UPDATE.sh` which doesn't use a tag for CCPPETMR_VM,and it's a tiny update, there's no reason to make a new VM for that reason only.

**Comment by johannesmayer** [comment]
Just to be sure: If you run the `download_PET_data.sh` script twice it also asks you if you want to replace the files when you unzip them!

Should I force an overwrite using `unzip -o ${filename}`?

**Comment by KrisThielemans** [comment]
good idea

**Comment by rijobro** [comment]
@johannesmayer just seen the diff from your PR, does that mean this is solved (commit ccc9e2a0e7bc410d4a4ce3d25b74086bcd94390d)?

**Comment by johannesmayer** [comment]
Yes I think so.
Running it the first time will download the files and unzip them if necessary. 
Running it twice will not download it again if it's already there and will unzip and overwrite what is already in the data folder without asking the user.

**Comment by KrisThielemans** [comment]
adressed in #23

**Comment by KrisThielemans** [comment]
closed by #23
